testsuite/ui/views/devel/__init__.py

```diff
@@ -137,7 +137,6 @@ def is_displayed(self):
         return (
             BaseDevelView.is_displayed.fget(self)
             and self.path in self.browser.url
-            and self.organization.is_displayed
```

Comment:
Why do you remove organization? Remove BaseDevelView instead ;)

Reply:
because signup view is different by signup flow for example via mail invitation there is no organization field and due to inheritance I need to have common base